proto/p2p/p2p.proto

```diff
@@ -303,6 +303,7 @@ message PushQuery {
   uint64 deadline = 3;
   bytes container = 4;
   EngineType engine_type = 5;
+  uint64 requested_height = 6;
```

Comment:
Should we populate requestedHeight last in the messages above then?

Reply:
The order doesn't actually matter... I felt like the code flowed more naturally with the order I used... The only reason the proto uses this order is because it is required for compatibility